openmls/src/credentials/mod.rs

```diff
@@ -77,8 +78,8 @@ pub enum CredentialType {
     Basic = 1,
     /// An X.509 [`Certificate`]
     X509 = 2,
-    /// A currently unknown credential.
-    Unknown(u16),
+    /// Another type of credential that is not in the MLS protocol spec.
+    Other(u16),
```

Comment:
That's a breaking change that needs to go into the changelog (unless we didn't have the unknown in the last release)

Reply:
I added that to the changelog